jdaviz/core/validunits.py

```diff
@@ -101,6 +91,44 @@ def create_flux_equivalencies_list(flux_unit, spectral_axis_unit):
     return sorted(units_to_strings(local_units)) + flux_unit_equivalencies_titles
 
 
+def create_sb_equivalencies_list(sb_unit, spectral_axis_unit):
+    """Get all possible conversions for flux from current flux units."""
+    if ((sb_unit in (u.count, u.dimensionless_unscaled))
+            or (spectral_axis_unit in (u.pix, u.dimensionless_unscaled))):
+        return []
+
+    # Get unit equivalencies. Value passed into u.spectral_density() is irrelevant.
+    try:
+        curr_sb_unit_equivalencies = sb_unit.find_equivalent_units(
+            equivalencies=u.spectral_density(1 * spectral_axis_unit),
+            include_prefix_units=False)
+    except u.core.UnitConversionError:
+        return []
+
+    locally_defined_sb_units = ['Jy / sr', 'mJy / sr', 'uJy / sr', 'MJy / sr', 'Jy / sr',
+                                'W / (Hz sr m2)',
+                                'eV / (Hz s sr m2)',
+                                'AB / sr'
+                                # 'erg / (s cm2 sr)',
+                                # 'erg / (s cm2 Angstrom sr)',
+                                # 'erg / (s cm2 Hz sr)',
+                                # 'ph / (Angstrom s sr cm2)',
+                                # 'ph / (s cm2 Hz sr)'
```

Comment:
Similar to my earlier comment, I'd rather us remove unsupported units than just commenting them out.
```suggestion
```